src/main/java/org/mskcc/cbio/oncokb/web/rest/MailsController.java

```diff
@@ -183,4 +189,60 @@ public TerminationEmailDTO getTerminationWarningEmail(@PathVariable Long company
     public void sendTerminationWarningEmail(@Valid @RequestBody TerminationEmailDTO terminationEmailDTO) throws MessagingException {
         this.mailService.sendEmail(terminationEmailDTO);
     }
+
+    @PostMapping("mails/termination-notification")
+    public void sendTerminationNotificationEmail(@Valid @RequestBody List<Long> companyIds) throws MessagingException {
+        List<CompanyDTO> allCompanies =  companyService.findAll();
```

Comment:
We don't have a service to fetch companies based on ids?

Reply:
As far as I can tell, [yes](https://github.com/oncokb/oncokb-public/blob/master/src/main/java/org/mskcc/cbio/oncokb/service/CompanyService.java).

Reply:
I think we should add one.